include/db/object.h

```diff
@@ -33,6 +35,8 @@ class LocalObject : public QEnableSharedFromThis<LocalObject> {
   Universe *db_;
   QString name_;
   QString comment_;
+  static std::atomic<uint64_t> static_id_;
+  uint64_t id_;
```

Comment:
Yes it will be required if we allow edition - however that is not exactly the case currently (application ui only allows to edit chunk comment - which I didn't even implement in this network communication) - I think it would be better to add it when we really gonna use it.